applications/tari_console_wallet/src/main.rs

```diff
@@ -219,6 +221,13 @@ fn main_inner() -> Result<(), ExitError> {
     result
 }
 
+fn get_password(config: &ApplicationConfig, cli: &Cli) -> Option<SafePassword> {
+    cli.password
+        .as_ref()
+        .or(config.wallet.password.as_ref())
```

Comment:
The logic was already in the code previously. But is it safe in case the user doesn't provide `passwor` via the cli, we are getting it from the `config.wallet` ?

Reply:
Previously, this logic was placed within `fn main_inner()`, file `applications/tari_console_wallet/stc/main.rs`. It might be potentially risky to use this function outside that given context.

Reply:
Looks ok to me - not sure exactly what the concern is but since the password bytes aren't moved around the stack (only the pointer Box<[u8]> is, not the contents of the pointer), and the function is private I don't think there is any additional risk.

Reply:
Perfect, thanks for the explanation @sdbondi ! :)